### DISCUSSION TITLE
A way to differentiate between lossy and lossless M4A files in joblist?

### DISCUSSION THREAD

**Discussion by cpuhog** [enzo1982/freac#312] [Help & Support]
I'm loading large playlists into fre:ac in order to transcode lossless ALAC (Apple lossless) files to MP3.  However, there are lossy AAC files mixed into these playlists that should not be transcoded.  I only want to process the lossless files.  The problem is both the ALAC and AAC files have the same file type: M4A.

There seems to be no field in the fre:ac joblist I can use to distinguish/sort between the two file types.  There are Format and CodecID fields in the file tags that could be used but fre:ac does not support those in the joblist.

Is there any extension or feature of fre:ac I can use to accomplish this?

Thanks.

**Comment by enzo1982** [comment]
The only way to distinguish between lossy and lossless .m4a files in the joblist is to look at the bitrate. The bitrate column can be enabled in the config dialog under _Interface->Joblist_.

For stereo content, AAC will usually have bitrates below 500 kbps (most of the time below 300 kbps) and ALAC above 600 kbps (most of the time above 800 kbps).

You can sort the joblist by bitrate by clicking on the column header. That should make it relatively easy to remove all the AAC files.

**Reply by cpuhog** [reply]
Okay, I'll try it that way.  Thanks for the reply.
